src/graphnet/data/sqlite_dataset.py

```diff
@@ -30,6 +30,9 @@ def __init__(
         node_truth: Optional[List[str]] = None,
```

Comment:
I think this file has moved due to #215. Could you merge this feature branch with `main` to make sure that we are not adding a second `sqlite_dataset.py` file, much like what happened with the I3Extractors, cf. the discussion with Kunal?